README.md

```diff
@@ -96,7 +96,7 @@ The following table gives an overview of the tools that we know about. If there
 | Julia          |                                                                                 |                                                                     | • [Bibliography.jl](https://github.com/Humans-of-Julia/Bibliography.jl)                    | • [Bibliography.jl](https://github.com/Humans-of-Julia/Bibliography.jl)                                                                             |
 | PHP            |                                                                                 |                                                                     |                                                                                 | • [bibtex-to-cff](https://github.com/monperrus/bibtexbrowser/)                                                                                                   |
 | Python         |                                                                                 | • [doi2cff](https://github.com/citation-file-format/doi2cff)        | • [cffconvert](#validation-heavy_check_mark)                                    | • [cffconvert](https://github.com/citation-file-format/cff-converter-python)<br>• [doi2cff](https://github.com/citation-file-format/doi2cff)                     |
-| R              |                                                                                 |                                                                     |                                                                                 | • [citation](https://cran.r-project.org/web/packages/citation/)<br>• [r2cff](https://github.com/ocbe-uio/RCFF)<br>• [handlr](https://github.com/ropensci/handlr) |
+| R              |                                                                                 |                                                                     |                                                                                 | • [citation](https://cran.r-project.org/web/packages/citation/)<br>• [r2cff](https://github.com/ocbe-uio/RCFF)<br>• [handlr](https://github.com/ropensci/handlr)<br>• [cffr](https://CRAN.R-project.org/package=cffr) |
```

Comment:
Hi @dieghernan, thanks for this PR! Happy to merge it once the workflows have run.
Just out of interest, is there a reason why you use this URL rather than the `web` url used by `citation` in the same cell (`https://cran.r-project.org/web/packages/citation/`)? Is this URL type resolved in other places than the browser (e.g., when pulling dependencies)?